fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/BaseObject.java

```diff
@@ -256,7 +256,7 @@ public void applyTx(IDatabaseAdapter target, ITransactionProvider tp, IVersionHi
     @Override
     public void applyVersion(IDatabaseAdapter target, IVersionHistoryService vhs) {
         if (vhs.applies(getSchemaName(), getObjectType().name(), getObjectName(), version)) {
-            logger.fine("Applying change [v" + version + "]: " + this.getTypeAndName());
+            logger.fine("Applying change [v" + version + "]: " + this.getTypeNameVersion());
```

Comment:
isn't this inclusive of type name and version already? version isn't necessary

Reply:
No, the logic requires we use version to distinguish the name - this is really important when creating the task list, which is used to apply the objects in parallel. Each version ends up being a task, but depends on the prior version so that they are applied in the correct order.